src/sdk/resources/auto_detect_compilers.xrc

```diff
@@ -22,16 +22,46 @@
 					<growablerows>0</growablerows>
 					<object class="sizeritem">
 						<object class="wxListCtrl" name="lcCompilers">
-							<style>wxLC_REPORT|wxLC_SINGLE_SEL</style>
+							<style>wxLC_REPORT|wxLC_SINGLE_SEL|wxVSCROLL|wxHSCROLL</style>
 						</object>
 						<flag>wxEXPAND</flag>
-						<minsize>460,240</minsize>
 					</object>
 					<object class="sizeritem">
-						<object class="wxButton" name="btnDefault">
-							<label>Set as default</label>
+						<object class="wxGridSizer">
+							<cols>1</cols>
+							<rows>3</rows>
+							<object class="sizeritem">
+								<object class="wxButton" name="btnSetDefaultCompiler">
+									<label>Set as default</label>
+									<pos>-1,-1</pos>
+									<size>-1,-1</size>
+								</object>
+								<flag>wxALIGN_CENTER_HORIZONTAL|wxALIGN_CENTER_VERTICAL</flag>
+								<border>5</border>
+							</object>
+							<object class="sizeritem">
+								<object class="wxRadioBox" name="rbCompilerShowOptions">
+									<label>Show</label>
+									<content>
+										<item>All</item>
+										<item>Detected</item>
+									</content>
+									<default>1</default>
+								</object>
+								<flag>wxALL|wxALIGN_CENTER_HORIZONTAL|wxALIGN_CENTER_VERTICAL</flag>
+								<border>5</border>
+							</object>
+							<object class="sizeritem">
+								<object class="wxCheckBox" name="cbShowCompilerPath">
+									<label>Show Path</label>
+								</object>
+								<flag>wxALL|wxALIGN_LEFT|wxALIGN_CENTER_VERTICAL</flag>
+								<border>5</border>
+							</object>
```

Comment:
Useless complexity. Please remove.

Reply:
This is needed if the if the wxStdDialogButtonSizer change stays otherwise the OK button moves to the very left.

Reply:
This comment is related to the radiobox and the checkbox, not about the alignment.
I want no checkbox and no radiobox.